assume/units/powerplant.py

```diff
@@ -91,16 +89,30 @@ def __init__(
             **kwargs,
         )
 
+        self.id = id
+        self.unit_operator = unit_operator
+        self.technology = technology
+        self.bidding_strategies = bidding_strategies
+        self.index = index
```

Comment:
I would like to remove these 5 lines - as this already happens in the call to super() and should not be done twice

Reply:
Oh, this was copilot...sorry, will remove it